docs/configuration/configuration.md

```diff
@@ -256,6 +256,11 @@ hetzner_sd_configs:
 http_sd_configs:
   [ - <http_sd_config> ... ]
 
+
+# List of IONOS service discovery configurations.
+ionos_sd_configs:
```

Comment:
This should be twice on the page, once for alerting and once for scraping.

Reply:
Thanks and fixed.